### PR TITLE
EditDetails.tsx localization (en) complete

### PR DESCRIPTION
**Pull request by DaveOrDavid**
## Description

English localization completed in EditDetails.tsx and translation.json

## Testing

Local visual confirmation plus successful local test.
---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/containers/UserListing.tsx [hunk 1]
```diff
@@ -167,9 +167,6 @@ function UserListing() {
               buttonText={t("UserListingDropdownMenuActions")}
               variant="outline"
             >
-              <MenuItem onSelect={changeRole} disabled={selected.length === 0}>
```
Comment (trietlu) [inline frontend/src/containers/UserListing.tsx:170]: @DaveOrDavid I don't think you meant to delete this block of code.  I think your code was not sync'ed (?)

**Review by trietlu — CHANGES_REQUESTED** [review]
File UserListing.tsx, lines 170-172 shouldn't be removed

File: frontend/src/containers/UserListing.tsx (hunk 1 above)
Comment (DaveOrDavid) [reply frontend/src/containers/UserListing.tsx:170]: Did a rebase prior to working so a mistake by me on removing this from merge conflicts. I'll change and resubmit

File: frontend/src/containers/EditDetails.tsx [hunk 2]
```diff
@@ -110,7 +115,9 @@ function EditDetails() {
                     id="topicAreaId"
                     name="topicAreaId"
                     label={settings.topicAreaLabels.singular}
-                    hint={`Select an existing ${settings.topicAreaLabels.singular.toLowerCase()}`}
+                    hint={`${t(
+                      "SelectExistingLeading"
```
Comment (ajmokotoff) [inline frontend/src/containers/EditDetails.tsx:119]: `t("SelectExistingLeading")` points to "***s**elect an existing". The s is lower case, perhaps we can just change the entry in the translation.json file? If we need it lower case in other places, maybe just make another entry?

**Review by ajmokotoff — COMMENTED** [review]
I found one very minor nitpick. Everything else looks really good. Great job David!

File: frontend/src/containers/EditDetails.tsx (hunk 2 above)
Comment (DaveOrDavid) [reply frontend/src/containers/EditDetails.tsx:119]: I can change that right now, give me few minutes

**Review by ajmokotoff — APPROVED** [review]

**Review by trietlu — APPROVED** [review]
lgtm

**Pull request merged**